lib/models/ObjectMD.js

```diff
@@ -1194,6 +1195,31 @@ class ObjectMD {
     getValue() {
         return this._data;
     }
+
+    /**
+     * Get x-amz-restore
+     *
+     * @returns {ObjectMDAmzRestore} x-amz-restore
+     */
+    getAmzRestore() {
+        return this._data['x-amz-restore'];
+    }
+
+    /**
+     * Set x-amz-restore
+     *
+     * @param {ObjectMDAmzRestore} value x-amz-restore object
+     * @returns {ObjectMD} itself
+     * @throws {Error} case of invalid parameter
+     */
+    setAmzRestore(value) {
```

Comment:
Any way to clear it after it's not needed anymore?

Reply:
Thanks, I agree this is needed. I added a way to clear it when the setter is used with an undefined (or falsy...) value. See https://github.com/scality/Arsenal/pull/1855/commits/3388de6fb6a8e66d6cdf3ac8f32d86cebeef1e34